tests/integration/test_examples.py

```diff
@@ -17,7 +17,7 @@ def test_policies(self, app):
 
     def test_scans(self, app):
         from examples.scans import example
-        example(app.session_name, app.session_file_output)
+        example(app.session_name, app.session_file_output, app.session_scan_target)
```

Comment:
It's fine to just pass `TenableIOTestConfig.get('scan_text_targets')` directly here. The other two methods are needed to generate test-session unique names.